### PR TITLE
Fix meta partial, edit links and mermaid

### PR DESCRIPTION
**Pull request by MarcelMue**
### What does this PR do?

This adds a meta page which is used for proper link formatting, mermaid and other convenience things.

### How does it look like?

No visible change but links like "edit this page" now redirect correctly.

### CODE REVIEW

**Comment by lyind** [comment]
Interesting. This kinda fixes google/docsy#1486 "locally".

**Comment by MarcelMue** [comment]
Indeed - some more stuff for mermaid in there as well though :)

**Comment by lyind** [comment]
Mermaid is cool, because it doesn't require any server-side support for rendering!

This site compares Mermaid VS PlantUML for various diagram types: https://www.dandoescode.com/blog/plantuml-vs-mermaid

I wonder if there's a converter for PlantUML to Mermaid syntax (tons of old diagrams).

Another options would be: https://github.com/mermaid-js/mermaid/issues/149

File: layouts/partials/page-meta-links.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{{ $editURL := printf "%s/edit/%s/content/%s" $gh_repo $gh_branch $pathFormatted }}
+{{ $issuesURL := printf "%s/issues/new?title=%s" $gh_repo (htmlEscape $.Title )}}
+<a href="{{ $editURL }}" target="_blank"><i class="fa fa-edit fa-fw"></i> {{ T "post_edit_this" }}</a>
+<a href="{{ $issuesURL }}" target="_blank"><i class="fab fa-github fa-fw"></i> {{ T "post_create_issue" }}</a>
+{{ if $gh_project_repo }}
+{{ $project_issueURL := printf "%s/issues/new" $gh_project_repo }}
+<a href="{{ $project_issueURL }}" target="_blank"><i class="fas fa-tasks fa-fw"></i> {{ T "post_create_project_issue" }}</a>
+{{ end }}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.5.2/mermaid.min.js"></script>
+<script>
```
Comment (lyind) [inline layouts/partials/page-meta-links.html:18]: Should we maybe load and initialize `mermaid` globally in a more central location?

Doing it in this partial seems OK, but somewhat hacky. (?)

Comment (MarcelMue) [reply layouts/partials/page-meta-links.html:18]: This is the solution we currently use in our intranet - so my main priority right now is to have parity. I don't know where a more central location would be, as the entrypoint could be any random page.

**Review by lyind — APPROVED** [review]
LGTM

**Pull request merged**